J1 - Accessibilité - RWD - UX/04 - Sujet Etude de cas/groupe4/www/css/main.css

```
@@ -0,0 +1,128 @@
+*{
+  font-family: 'Montserrat', sans-serif;
+  align-items: center;
+}
+
+html {
+  background: rgb(176, 202, 209);
```

je prefre que le fond soit rouge
```suggestion
  background: red;
```

---

Non je suis pas d'accord